internal/helm/controller/reconcile.go

```diff
@@ -316,6 +316,16 @@ func (r HelmOperatorReconciler) Reconcile(request reconcile.Request) (reconcile.
 	}
 
 	log.Info("Reconciled release")
+	reason := types.ReasonUpgradeSuccessful
+	if expectedRelease.Version == 1 {
+		reason = types.ReasonInstallSuccessful
+	}
+	status.SetCondition(types.HelmAppCondition{
+		Type:    types.ConditionDeployed,
+		Status:  types.StatusTrue,
+		Reason:  reason,
+		Message: expectedRelease.Info.Notes,
```

Comment:
we need to check if the info is not nil. Such as : https://github.com/operator-framework/operator-sdk/blob/ed95c5af3e937a71e87bca6348bd8ad3a965e7ac/internal/helm/controller/reconcile.go#L212-L215

Reply:
Thanks for the good catch.